lib/tasks/evm_import_export_options.rb

```diff
@@ -0,0 +1,100 @@
+# @class EvmExportImportOptions
+# @brief Simple class to parse options passed in to a rake task.
+class EvmImportExportOptions
```

Comment:
This class can probably be replaced by Trollop using the rake task options like https://github.com/ManageIQ/manageiq/blob/02c1f12c0f13efeb9a3faaa8c5031a3fd5a20410/lib/tasks/evm_dba.rake#L73